### PR TITLE
Adding a module for GetOrganelle

### PR DESCRIPTION
**Pull request by erinyoung**
## PR checklist

Closes https://github.com/nf-core/modules/issues/4444

This PR is for using GetOrganelle with get_organelle_from_reads.py, but that required get_organelle_config.py. Therefore, there are two modules that are part of this PR. 

I created these files and tested them in a gitpod environment, for which singularity doesn't work and conda (for whatever reason) wasn't working. I'm hopeful that these modules work in those environments.

I have not added any files to the test data. Instead, I attempted to use the human data - which generates empty files. I am trying to not add additional files to the test data repo, but I can if the reviewers think it's necessary. 

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [X] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/getorganelle/config/main.nf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    get_organelle_config.py \\
+        -a ${organelle_type} \\
+        --config-dir getorganelle
```
Comment (ggabernet) [inline modules/nf-core/getorganelle/config/main.nf:25]:
```suggestion
    get_organelle_config.py \\
        -a ${organelle_type} \\
        --config-dir getorganelle $args
```

**Review by ggabernet — COMMENTED** [review]
I have a couple of comments but otherwise looks great to me!

File: modules/nf-core/getorganelle/config/meta.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: "getorganelle_config"
+description: Downloads databases
```
[the remark this answers is not in the document]
Comment (erinyoung) [reply modules/nf-core/getorganelle/config/meta.yml]: Done!

File: modules/nf-core/getorganelle/config/main.nf (hunk 1 above)
Comment (erinyoung) [reply modules/nf-core/getorganelle/config/main.nf]: Sorry! I forgot to add it! I've decided to put it on a different line, though.

**Review by ggabernet — APPROVED** [review]
Looks good to me now!

**Pull request merged**